sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1724,12 +1724,18 @@ pi_result ExecCGCommand::SetKernelParamsAndLaunch(
 void DispatchNativeKernel(void *Blob) {
```

Comment:
By looking at the code I wonder why `Blob` could not be more carefully typed at the first place instead of all this `******` spaghetti code that gives mitigated trust in the code... :-/